irods_testing_environment/odbc_setup.py

```diff
@@ -238,6 +238,22 @@ def configure_odbc_driver_centos_7_mysql_57(csp_container, odbc_driver):
     configure_mysql_odbc_driver(csp_container, os.path.abspath(odbc_driver))
 
 
+def configure_odbc_driver_centos_7_mysql_8029(csp_container, odbc_driver):
+    """Configure ODBC driver for mysql 8.0 on centos 7.
+
+    Argument:
+    csp_container -- docker container on which the iRODS catalog service provider is running
+    odbc_driver -- path to local archive file containing the ODBC driver package
+    """
+    if not odbc_driver:
+        odbc_driver = download_mysql_odbc_driver(
+            #'https://dev.mysql.com/get/Downloads/Connector-ODBC/8.0/mysql-connector-odbc-8.0.31-1.el7.x86_64.rpm')
+            #'https://dev.mysql.com/get/Downloads/Connector-ODBC/8.0/mysql-connector-odbc-8.0.31-linux-glibc2.27-x86-64bit.tar.gz')
```

Comment:
Any reason to keep these around? It's fine with me, just curious. Maybe worth a comment.

Reply:
I don't think we need them. They are leftovers for when I was trying to make it work.

At some point, we'll need to bump the version of the database. We can revisit the drivers when that time comes.

Reply:
Removed the lines.